tools/combine-files/combine_files.py

```
@@ -1311,6 +1312,230 @@ def _get_relative_path_or_original(config_file, file_path_root):
+                    # This may fail for clean files, so catch the exception
+                    try:
+                        staged_diff = repo.head.commit.diff(
+                            repo.index, paths=str(file_path_in_repo)
+                        )
+                        if staged_diff:
+                            status = "staged"
+                        # If no unstaged and no staged diff, file is clean
+                    except git.exc.GitCommandError:
+                        # diff(repo.index) can fail for clean files, assume clean
+                        status = "clean"
+        except (git.exc.GitCommandError, ValueError) as e:
```

Ruff (0.14.8)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)